static/js/command_sequencer.js

```diff
@@ -220,6 +228,7 @@ function execute_sequence(button) {
     } else {
         disable_buttons(`${BUTTON_ID['all_execute']},${BUTTON_ID['reload']}`, true);
         disable_buttons(`${BUTTON_ID['abort']}`, false);
+        hide_execution();
```

Comment:
This should likely be `display_execution(`${seq_module_name/${seq_name}`);`, or it hides the progress bar for sequences with no arguments.

Reply:
Thanks, I didn't pick that up in testing. I think I added the `hide_execution()` call in two places rather than just the later one 🤦